tools/keycloak-user-initializer/src/main/resources/data/tco-user-list.csv

```diff
@@ -0,0 +1,7 @@
+﻿Email,Realm Administrator,admin-judicial-justice,admin-vtc-staff,judicial-justice,vtc-staff
+colm.ohiggins@gov.bc.ca,TRUE,TRUE,TRUE,TRUE,TRUE
+phil.bolduc@gov.bc.ca,TRUE,TRUE,TRUE,TRUE,TRUE
+marti.stewart@gov.bc.ca,TRUE,TRUE,TRUE,TRUE,TRUE
+shark.wong@gov.bc.ca,TRUE,TRUE,TRUE,TRUE,TRUE
+sylvia.restall@gov.bc.ca,TRUE,TRUE,TRUE,TRUE,TRUE
+lorraine.dame@gov.bc.ca,TRUE,TRUE,TRUE,TRUE,TRUE
```

Comment:
As a public repo, should these emails be in GIT?

Reply:
Oh you're absolutely right! They should not be in the GIT repo. I forgot removing this from the CSV file. I will remove all the data except the headers and commit again. Thanks for catching this :)

Reply:
I would suggest you remove the file and rebase to purge the git history.